emporous-community/sample-client-python
# feat: initial implementation of sample-client-python

Initial commit for sample python client

# How to test

1. Start up server 
`uor-client-client serve test.sock`
2. Setup and run client
```
pipenv shell
pipenv install
export EMPOROUS_SOCKET_ADDRESS=test.sock /. Default to /var/run/emporous.sock
python3 sample_client.py push test/workspace localhost:5001/test:latest
python3 sample_client.py pull localhost:5001/test:latest testme
// This test attribute filter. The attributes pattern is by the gRPC client in this case
python3 sample_client.py pull localhost:5001/test:latest mytestdir --attributes='{"images": True}'
```

Should be merged with https://github.com/emporous/emporous-go/pull/110.
Signed-off-by: Jennifer Power <barnabei.jennifer@gmail.com>
